TextArea component (#686)
looker-open-source/components: packages/www/src/documentation/components/forms/fields.mdx

<FieldText name="requiredField" label="A required field" required />
```

## FieldTextArea

<Props of="FieldTextArea" />

> The `<FieldTextArea />` component is composed of an `<TextArea />` component and a `<Label />` component. Using `<FieldTextArea />` allows for rendering validation messages.

Reviewer: ```suggestion
The `<FieldTextArea />` component is composed of `<TextArea />` and `<Label />` components. Using `<FieldTextArea />` allows for rendering validation messages.
```

```jsx
<FieldTextArea

>   label="FieldTexArea Label"

Reviewer: Label has a typo: `FieldTexArea` is missing the `t` (should be `FieldTextArea`)
